h5p-bildetema/src/components/CollectionsController/MultiLanguageWord/MultiLanguageWord.module.scss

```diff
@@ -40,6 +40,51 @@
   line-height: 1;
 }
 
+.button {
+  all: unset;
+  position: absolute;
+  right: 0.5rem;
+  z-index: $fav-language-selector-z-index;
+  top: 0.5rem;
+
+  border-radius: 50%;
+  padding: 0.5rem;
+  background-color: $beige;
+  cursor: pointer;
+
+  &:hover {
+    background-color: $burgundy;
+    fill: white;
+  }
```

Comment:
Svgs can leave a spacing after themselves because they are inline-blocks. This can be avoided by using `display: block;` for instance on the svg element. There are also other solutions to avoid the spacing.
<img width="470" alt="Skjermbilde 2024-05-16 kl  15 20 02" src="https://github.com/Tietoevry-Create/h5p-bildetema/assets/35261194/7261c580-2ba0-44c0-8994-d3fa238024ae">

Reply:
Also noticed that we use a kebab menu icon (vertical dots) on the main collection page while here we have a meatball menu icon (horizontal dots). Maybe we should use keep to using one? What do you think would be the best? 
Example with kebab icon in a collection: 
<img width="367" alt="Skjermbilde 2024-05-16 kl  15 37 15" src="https://github.com/Tietoevry-Create/h5p-bildetema/assets/35261194/23afe94f-992e-4ac6-97a6-292dfb0adf16">

Example with hamburger icon on the main collection page:
<img width="810" alt="Skjermbilde 2024-05-16 kl  15 40 25" src="https://github.com/Tietoevry-Create/h5p-bildetema/assets/35261194/76d8e1af-8cd6-4bbf-b303-bf7769602d97">

Reply:
Yes 👌

Reply:
Updated